### PR TITLE
[Windows] Merge several toolset provisioners into single Configure-Toolset script

### PR DESCRIPTION
**Pull request by vmapetr**
# Description
This PR merges `Install-Boost` and `Install-Go` provisioners into single `Configure-Toolset.ps1` script.

**Details:**
`Configure-Toolset.ps1` is used to prepare system environment for toolset tools such as Boost, Go and Python, and set environment variables for default tool versions. This PR also moves toolset.json config files into separate folder `toolsets` in image template root.

#### Related issue:
 Toolset improvements for Windows images #687 

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/win/scripts/ImageHelpers/InstallHelpers.ps1 [hunk 1]
```diff
@@ -358,6 +358,44 @@ function Get-ToolsetContent {
     ConvertFrom-Json -InputObject $toolsetJson
 }
 
+Function Get-ToolsetToolFullPath
+{
+    param
+    (
+        [string] $Name,
+        [string] $Version,
+        [string] $Arch
```
[the remark this answers is not in the document]
Comment (vmapetr) [reply images/win/scripts/ImageHelpers/InstallHelpers.ps1]: Yep. Thanks!

File: images/win/scripts/Installers/Configure-Toolset.ps1 [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        )
+    }
+    Boost = @{
+        variableTemplate = "BOOST_ROOT_{0}_{1}_{2}"
+    }
+    go = @{
+        pathTemplates = @(
+            "{0}\bin"
+        )
+        rootVariableName = "GOROOT"
```
Comment (maxim-lobanov) [inline images/win/scripts/Installers/Configure-Toolset.ps1:49]: May be `defaultVariable` to be consistent with Ubuntu.

Comment (vmapetr) [reply images/win/scripts/Installers/Configure-Toolset.ps1]: Done

**Pull request merged**